### PR TITLE
Operator v1.5.0 release

### PR DESCRIPTION
**Pull request by zlogic**
Prepared release notes and updated documentation for the new Mendix Operator version.

### CODE REVIEW

**Comment by MarkvanMents** [comment]
@zlogic 
Hi Dmitrii,
Thanks for the Pull Request.
With these PRs it would be useful to have a target release date (or immediate release) so that I can make sure I look at it in time and am available to help with publishing.
Can you give an indication when this PR is likely to be published?
Thanks.

**Comment by zlogic** [comment]
@MarkvanMents We were planning to release it today or perhaps tomorrow (September 10).

I'll add a note to future PRs mentioning when's the intended release date.

File: content/developerportal/deploy/private-cloud-cluster.md [hunk 1]
```diff
@@ -399,6 +423,98 @@ When the namespace is configured correctly, its status will become **Connected**
...
+apiVersion: privatecloud.mendix.com/v1alpha1
+kind: OperatorConfiguration
+spec:
+  # Endpoint (Network) configuration
+  endpoint:
+    # Endpoint type: ingress or openshiftRoute
+    type: ingress
+    # Ingress configuration: used only when type is set to ingress
+    ingress:
+      # Optional, can be ommited: annotations which should be applied to all Ingress Resources
```
Comment (MarkvanMents) [inline content/developerportal/deploy/private-cloud-cluster.md:459]: I won't change this here, as it is probably in the original file, but it should be `omitted`

Comment (zlogic) [reply content/developerportal/deploy/private-cloud-cluster.md]: Thanks for noticing - the original file doesn't have any comments, so I've pushed a new commit to fix this.

**Comment by MarkvanMents** [comment]
I'll prioritize this. Hope to have it reviewed by 16:00.

File: content/releasenotes/developer-portal/deployment.md [hunk 2]
```diff
@@ -10,6 +10,23 @@ These release notes cover changes to [Mendix Cloud](/developerportal/deploy/mend
...
+#### Mendix for Private Cloud — Mendix Operator v1.5.0 and Mendix Gateway Agent v1.4.0
+
+* We added an option to configure TLS options per environment, overriding the default namespace settings
+* OpenShift Routes now support all features which were previously only available in Ingress endpoints
+* We now only monitor the selected endpoint type (OpenShift Routes or Ingess), removing an error message which appeared when starting the operator in a non-OpenShift cluster
+* We have added several features which make it possible to use [cert-manager](https://cert-manager.io/) with Ingress — this allows you to have TLS certificates generated on-demand
+* We added an option to specify an environment's TLS certificate, or load it from a Kubernetes secret
+* When TLS is enabled, Mendix for Private Cloud Portal will now display the App URL with an https:// prefix
+* We have fixed an issue with connecting to PostgreSQL with TLS and will use encryption by default
+* We have fixed an issue with an incorrect _failed to create role_ error message for PostgreSQL databases, which in some cases was caused by connection issues
```
Comment (MarkvanMents) [inline content/releasenotes/developer-portal/deployment.md:24]: @zlogic 
Hi Dmitrii,
Were there any tickets for these fixes, or were they discovered internally.

**Review by MarkvanMents — APPROVED** [review]
@zlogic 
Hi Dmitrii,
Thanks for this documentation - it looks very comprehensive and only needed some minor changes.
Let me know when you are ready to release.

File: content/releasenotes/developer-portal/deployment.md (hunk 2 above)
Comment (zlogic) [reply content/releasenotes/developer-portal/deployment.md]: @MarkvanMents Indeed, there was a ticket for the _PostgreSQL with TLS_ issue. I've added a commit mentioning the ticket number,

**Comment by zlogic** [comment]
Hi @MarkvanMents, can you please release the updated documentation now? Everything else is ready on our side.

**Comment by MarkvanMents** [comment]
Published

**Comment by MarkvanMents** [comment]
@zlogic : https://docs.mendix.com/releasenotes/developer-portal/deployment

**Comment by zlogic** [comment]
@MarkvanMents Thank you very much!

**Pull request merged**